lib/resources/storage/unsupported_storage.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/cupertino.dart';
+
+class FireStorageService extends ChangeNotifier {
+  FireStorageService._();
+
+  FireStorageService();
+
+  static Future<dynamic> loadImage(BuildContext context, String image) async {
+    // ignore: only_throw_errors
```

Comment:
*Once again*, don't just ignore warnings. The warning in AS points you to the documentation link where you can see what the good practice is.
https://dart-lang.github.io/linter/lints/only_throw_errors.html